### ISSUE TITLE
Design Image Entity in D8

### ISSUE THREAD

**Issue by dannylamb** [Islandora-CLAW/CLAW#339]
Depends on #334 and #335.

Do not attempt this until those have been resolved.  Outcome is a plan for implementation.  Like the other tickets, think in terms of classes and base fields.  What would the annotation look like?

How does IIIF compatability come into play?

**Comment by dannylamb** [comment]
Closing since sprint is over.  We can open another ticket to 'revisit' this concept later if required.
